specs/text-field/text-field.md

```diff
@@ -0,0 +1,142 @@
+# Text Field
+
+## Overview
+
+An implementation of a generic input for textual values as a form-connected web-component.
+
+### Use Cases
+
+Used anywhere an author might otherwise use:
+- input[type="text"]
+- input[type="email"]
+- input[type="password"]
+- input[type="tel"]
+- input[type="url"]
+
+### Features
+
+1. Form Association where supported
```

Comment:
I'm curious if you'll really want to use the form-association here because you'll actually be rendering an input element into the shadow DOM

Reply:
Gotcha - so if we have an input being rendered within the shadow DOM it will still function as an input and submit with the form?

Reply:
no longer using form association